bugbug/bugzilla.py

```diff
@@ -135,20 +135,22 @@ def download_bugs(bug_ids, products=None, security=False):
     print(f'Loaded {old_bug_count} bugs.')
 
     new_bug_ids = sorted(list(new_bug_ids))
-
     total_downloaded = 0
-    chunks = (new_bug_ids[i:(i + 500)] for i in range(0, len(new_bug_ids), 500))
-    for chunk in chunks:
-        new_bugs = _download(chunk)
 
-        total_downloaded += len(new_bugs)
+    chunks = (new_bug_ids[i:(i + 500)] for i in range(0, len(new_bug_ids), 500))
+    with tqdm(total=len(new_bug_ids)) as pbar:
+        for chunk in chunks:
+            new_bugs = _download(chunk)
 
-        print(f'Downloaded {total_downloaded} out of {len(new_bug_ids)} bugs')
+            total_downloaded += len(new_bugs)
```

Comment:
We no longer need `total_downloaded`, as the progress bar is going to show the progress